suite-native/module-device-settings/src/screens/DeviceAuthenticitySuccessScreen.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, PictogramTitleHeader, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Screen, ScreenHeader, useNavigateToInitialScreen } from '@suite-native/navigation';
+
+export const DeviceAuthenticitySuccessScreen = () => {
+    const navigateToInitialScreen = useNavigateToInitialScreen();
+
+    return (
+        <Screen
+            header={<ScreenHeader closeActionType="close" closeAction={navigateToInitialScreen} />}
+        >
+            <VStack flex={1} spacing="sp32">
+                <VStack flex={1} spacing="sp40" justifyContent="center">
+                    <PictogramTitleHeader
+                        variant="success"
+                        title={<Translation id="moduleDeviceSettings.authenticity.success.title" />}
+                        titleVariant="titleMedium"
+                        subtitle={
+                            <Translation id="moduleDeviceSettings.authenticity.success.subtitle" />
+                        }
+                    />
+                </VStack>
```

Comment:
Wouldn't this be sufficient?
```suggestion
                <PictogramTitleHeader
                    variant="success"
                    title={<Translation id="moduleDeviceSettings.authenticity.success.title" />}
                    titleVariant="titleMedium"
                    subtitle={
                        <Translation id="moduleDeviceSettings.authenticity.success.subtitle" />
                    }
                />
```

Reply:
BEFORE:
![Screenshot_2025-05-12-14-28-23-091_io trezor suite debug (1)](https://github.com/user-attachments/assets/26da4e54-21e9-48d9-93cc-f2b0c8512d5a)

AFTER:
![Screenshot_2025-05-12-14-30-43-315_io trezor suite debug (1)](https://github.com/user-attachments/assets/d08a1ba7-c335-4d18-bd97-37e228aa8731)


But the spacing is not used here, so I removed that efd150bc447390b1bf1ad6cd1a848aa70cc38bb4